tests/typescript_conditional_types/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -25,6 +33,16 @@ type DeepReadonlyObject<T> = {
   readonly [P in NonFunctionPropertyNames<T>]: DeepReadonly<T[P]>
 };
 
+type TypeName<T> = T extends string
+  ? "string"
+  : T extends number
+  ? "number"
+  : T extends boolean
+  ? "boolean"
+  : T extends undefined
+  ? "undefined"
+  : T extends Function ? "function" : "object";
```

Comment:
Unrelated to this PR: We should force breaking nested TS type ternaries just like normal ternaries.